src/pages/waypoint/tutorials/[...tutorialSlug]/index.tsx

```diff
@@ -20,9 +20,16 @@ export function WaypointTutorialPage({
 
 export async function getStaticProps({
   params,
-}): Promise<{ props: TutorialPageProps }> {
+}): Promise<GetStaticPropsResult<TutorialPageProps>> {
   const product = waypointData as LearnProductData
-  return await getTutorialPageProps(product, params.tutorialSlug)
+  const data = await getTutorialPageProps(product, params.tutorialSlug)
```

Comment:
This is a _super_ minor nitpick, but noticed:
- in `getTutorialPageProps` we call the return value within the function `props`, and assign it to `const data`
- in `getCollectionPageProps` we call the return value within the function `data`, and assign it to `const props`
I think consistently using `props` might be more clear, but still feels pretty clear as-it so definitely feel free to ignore 👍

Reply:
Good call, I'll adjust to use `props`! 👌